libs/shared/type-site/src/lib/i-component.ts

```diff
@@ -96,6 +96,14 @@ export interface IComponentStyle {
   overrides?: IComponentStyleOverrides
 }
 
+export interface IReusableComponentData {
+  // Reusable component id
+  id: string
+  // Inputs override
+  // Key: input key, value: input `is`
+  inputs?: Record<string, unknown>
```

Comment:
Oh, I didn't notice this when I wrote one of my previous comments. Is there a reason for using this, instead of considering the component's existing `inputs` as overrides?

Reply:
Sorry for the late reply!
No, I think using the component's existing `inputs` as overrides is a lot better than my previous implementation. I'm not sure why I did that. 🤔

Reply:
Got it! There's still some work to do and details to figure out. I'll leave another comment about what I think the remaining tasks are.